src/Admin/AdminInterface.php

```diff
@@ -678,10 +683,14 @@ public function getListMode();
 //    public function showMosaicButton(bool $isShown): void;
 
     /*
-     * Checks if a filter type is set to a default valu
+     * Checks if a filter type is set to a default value
+     *
+     * @param string $name
+     *
+     * @return bool
      */
 //    NEXT_MAJOR: uncomment this method in 4.0
-    // public function isDefaultFilter(string $name): bool;
+    // public function isDefaultFilter($name): bool;
```

Comment:
I think we should keep the argument declaration.

Reply:
Fixed